docs/scripting/language/Style.md

```diff
@@ -0,0 +1,216 @@
+---
+id: Style
+title: "Pawn Style Guide"
+description: A short guide on the generally accepted naming conventions and other aspects of Pawn source code to aid easier communication of intent and streamline debugging and sharing of code.
+---
+
+This document is a short guide on the generally accepted naming conventions and other aspects of Pawn source code to aid easier communication of intent and streamline debugging and sharing of code.
+
+See also:
+
+- [Modern Pawn](https://github.com/Southclaws/sampctl/wiki/Modern-Pawn)
+- [Pawn Package](https://github.com/Southclaws/sampctl/wiki/Packages)
+
+## Terminology
+
+### Modules
+
+A **Module** is a single Pawn source file, with an `.inc` extension.
```

Comment:
I've barely seen this on other places (other than samp discord and samp forums), calling include files `modules`; cause in the end they're just some files that can be included and can be ***added*** as part of the ***code*** itself, not the whole environment.

I've always thought of a module as something like what we call a filterscript in samp scene, a totally independent part of the environment that can be added or removed individually without any problem and without doing changes to other part of your code

Reply:
Tend to agree, include files in Pawn might be completely independent in some cases but overall act as C-like header files. I think calling them includes (or include files) is more straightforward

Reply:
Yeah I had trouble justifying this one tbh.

Module has more semantic meaning in terms of privacy, exposure and structure.

Reply:
We can do a new PR for a better definition.